packages/babel-plugin/src/babel-plugin.ts

```diff
@@ -295,6 +296,19 @@ export default declare<State>((api) => {
         path: NodePath<t.TaggedTemplateExpression> | NodePath<t.CallExpression>,
         state: State
       ) {
+        if (
```

Comment:
should this check that `cloneElement` is imported from `react` too?

Reply:
Yes definitely and also should check for aliased imports

```
import { cloneElement as foo } from 'react';
```

Reply:
Edit: I added support for alias and member expressions in the latest commit